pkg/client-sdk/go.mod

```diff
@@ -5,6 +5,8 @@ go 1.23.1
 replace github.com/btcsuite/btcd/btcec/v2 => github.com/btcsuite/btcd/btcec/v2 v2.3.3
 
 require (
+	github.com/ark-network/ark/api-spec v0.0.0-20241111144357-cd502f3dac28
+	github.com/ark-network/ark/common v0.0.0-20241111144357-cd502f3dac28
```

Comment:
DO NOT run go mod tidy to install deps, use go work sync instead.

ark's deps must not show up in any go mod.

Reply:
hmm, i run go work sync in root